src/UKHO.ERPFacade.PublishPriceChange.WebJob/appsettings.json

```diff
@@ -28,5 +28,12 @@
     },
     "AzureStorageConfiguration": {
         "ConnectionString": ""
-    }
+    },
+    "EnterpriseEventServiceConfiguration": {
+        "ServiceUrl": "",
+        "PublishEndpoint": "api/events",
+        "ClientId": "",
+        "PublisherScope": ".default"
+    },
+    "ApplicationUri": "https://erp.ukho.gov.uk"
```

Comment:
Can we move this under EnterpriseEventServiceConfiguration and rename it as ErpPublishEventSource ?

Reply:
Renamed. Thanks.